content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -19,7 +19,7 @@ This document catalogs the communication paths between the control plane (really
 {{% capture body %}}
 
 ## Node to Control Plane
-All communication paths from the nodes to the control plane terminate at the apiserver (none of the other master components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
+All communication paths from the nodes to the control plane terminate at the apiserver (none of the other control plane components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (6443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
```

Comment:
How about:
```suggestion
All communication paths from the nodes to the control plane terminate at the apiserver (none of the other control plane components are designed to expose remote services). In a typical deployment, the apiserver listens for remote connections on a secure HTTPS port (443 or 6443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
```
?

I'm reluctant to completely drop mentioning use port 443 for serving the Kubernetes API; from my personal experience that is a very common configuration. It's also the usual port for HTTPS outside of Kubernetes.

Reply:
I'm not sure if it's good to list two ports when talking about a typical deployment, which sounds a little confusing to me. And how do we handle inconsistent document: https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/#transport-security, which says
> In a typical Kubernetes cluster, the API serves on port 6443.

Reply:
Mmm. Let's reach consensus and document one consistent port number for HTTPS (as in API server HTTPS).

Reply:
AFAIK, ~100% of installations use 443 as the actual port that api users connect to. IIRC, I made 6443 the default for the flag back in the day, because we had to run an nginx front proxy (!). People using 6443 as the port almost certainly run a proxy that listens at 443.

```suggestion
Kubernetes has a "hub-and-spoke" API pattern. All API usage from nodes (or the pods they run) terminate at the apiserver (none of the other control plane components are designed to expose remote services). The apiserver is configured to listen for remote connections on a secure HTTPS port (typically 443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
```

Reply:
Thanks @lavalamp for sharing it and the suggestion. Updated per your suggestion.